core/models.py

```
@@ -882,77 +691,660 @@ def title_suggestion_schema(self):
+
+            deps = BlogPostGenerationContext(
+                project_details=self.project.project_details,
+                title_suggestion=self.title_suggestion_schema,
+                project_pages=project_pages,
+                content_type=self.content_type,
+                project_keywords=project_keywords,
+            )
+
+            result = run_agent_synchronously(
+                agent,
+                "Please generate the full blog post content following the provided structure outline.",  # noqa: E501
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)